### PR TITLE
docs(CONTRIBUTING.md): fix some typos

### PR DESCRIPTION
**Pull request by mathiasbynens**
As someone who's new to the project, I read the contributing guide and spotted some things that confused me. Hopefully this patch helps to reduces the confusion for others!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -232,42 +233,44 @@ See [Debugging Tips](README.md#debugging-tips) in the readme.
 
 # For Project Maintainers
 
-## Releasing to NPM
+## Releasing to npm
+
+Releasing to npm consists of the following phases:
 
-Releasing to NPM consists of 3 phases:
```
Comment (mathiasbynens) [inline CONTRIBUTING.md:237]: I'm assuming the number wasn't updated when the fourth phase was added.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -232,42 +233,44 @@ See [Debugging Tips](README.md#debugging-tips) in the readme.
...
     1. Bump `package.json` version to `-post` version and send a PR titled `'chore: bump version to vXXX.YYY.ZZZ-post'` ([example](https://github.com/GoogleChrome/puppeteer/commit/d02440d1eac98028e29f4e1cf55413062a259156))
         - **NOTE**: make sure to update the "released APIs" section in the top of `docs/api.md`.
         - **NOTE**: no other commits should be landed in-between release commit and bump commit.
 
-## Updating NPM dist tags
+## Updating npm dist tags
+
+For both `puppeteer` and `puppeteer-core` we maintain the following npm tags:
 
-For both `puppeteer` and `puppeteer-firefox` we maintain the following NPM Tags:
```
Comment (mathiasbynens) [inline CONTRIBUTING.md:262]: I think `puppeteer-firefox` was a typo here, and `puppeteer-core` is what was meant.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -232,42 +233,44 @@ See [Debugging Tips](README.md#debugging-tips) in the readme.
...
 # For Project Maintainers
 
-## Releasing to NPM
+## Releasing to npm
+
+Releasing to npm consists of the following phases:
 
-Releasing to NPM consists of 3 phases:
 1. Source Code: mark a release.
     1. Bump `package.json` version following the SEMVER rules and send a PR titled `'chore: mark version vXXX.YYY.ZZZ'` ([example](https://github.com/GoogleChrome/puppeteer/commit/808bf8e5582482a1d849ff22a51e52024810905c)).
```
Comment (mathiasbynens) [inline CONTRIBUTING.md:241]: Drive-by question: are PRs like this created manually? I'm asking because the `<!-- GEN:empty-if-release -->`-style comments imply otherwise.

Comment (aslushnikov) [reply CONTRIBUTING.md:241]: These PRs are created manually, but the `<!-- GEN:empty-if-release -->` are updated with our [markdown preprocessor](https://github.com/GoogleChrome/puppeteer/tree/master/utils/doclint).

File: CONTRIBUTING.md (hunk 2 above)
Comment (aslushnikov) [reply CONTRIBUTING.md:262]: thanks! That's right!

**Review by aslushnikov — APPROVED** [review]

**Comment by aslushnikov** [comment]
Thanks!

File: CONTRIBUTING.md (hunk 3 above)
Comment (mathiasbynens) [reply CONTRIBUTING.md:241]: Thanks! I'll submit a follow-up CL that explicitly calls out that one needs to run `npm run doc` after bumping the `package.json` version number.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -232,42 +233,44 @@ See [Debugging Tips](README.md#debugging-tips) in the readme.
...
     2. Make sure the PR passes **all checks**.
         - **WHY**: there are linters in place that help to avoid unnecessary errors, e.g. [like this](https://github.com/GoogleChrome/puppeteer/pull/2446)
     3. Merge the PR.
     4. Once merged, publish release notes using the "create new tag" option.
```
Comment (mathiasbynens) [inline CONTRIBUTING.md:245]: > the "create new tag" option

What is this referring to exactly? On https://github.com/GoogleChrome/puppeteer/releases I see a "Draft a new release" button -- is that what is meant? 

Is there some tool or `git log`-based command-line script you're using for the release notes ([example](https://github.com/GoogleChrome/puppeteer/releases/tag/v1.19.0))?

Comment (mathiasbynens) [reply CONTRIBUTING.md:245]: https://github.com/GoogleChrome/puppeteer/pull/4940
